### PR TITLE
Nikita/notifications

### PR DESCRIPTION
**Pull request by npelenskii**
Add correct usage of indexed db. Notifications MVP

### CODE REVIEW

File: src/firebase.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ export const onMessageListener = async () => {
   const messagingResolve = await messaging;
   try {
     const message = onMessage(messagingResolve, payload => {
-      console.log(payload);
+      //console.log(payload);
```
Comment (KMyroslav) [inline src/firebase.js:57]: Consider removing comments

File: src/redux/notifications/notificationSlice.js [hunk 2]
```diff
@@ -17,6 +17,10 @@ const notificationSlice = createSlice({
       state.notificationList.unshift(payload);
       state.newNotification = payload;
     },
+    addMultipleNotification: (state, { payload }) => {
+      state.notificationList = state.notificationList.concat(payload);
+      //state.newNotification = payload[0];
```
Comment (KMyroslav) [inline src/redux/notifications/notificationSlice.js:22]: Maybe this one too? if you don't need it in the future

File: src/firebase.js (hunk 1 above)
Comment (npelenskii) [reply src/firebase.js:57]: Okey)

**Review by KMyroslav — APPROVED** [review]

**Review by Bogd1y — APPROVED** [review]

File: public/firebase-messaging-sw.js [hunk 3]
```diff
@@ -22,9 +22,23 @@ const firebaseConfig = {
...
 const messaging = firebase.messaging();
 
-messaging.onBackgroundMessage(payload => {
-  //console.log('Received background message ', payload);
+let channel;
+
+const getChannel = () => {
+  let channel = new BroadcastChannel('sw-messages');
+  return channel;
+};
```
Comment (AntonDeXy) [inline public/firebase-messaging-sw.js:30]:
```suggestion
const getChannel = () => {
  const channel = new BroadcastChannel('sw-messages');
  return channel;
};
```

File: src/utils/notification/notificationChannel.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+let channel;
+
+const getChannel = () => {
+  let channel = new BroadcastChannel('sw-messages');
+  return channel;
+};
+
+export const notificationChannel = {
+  getInstance: () => {
+    if (channel === undefined || channel === null) {
```
Comment (AntonDeXy) [inline src/utils/notification/notificationChannel.js:10]: will if (!channel) fit our needs

Comment (npelenskii) [reply src/utils/notification/notificationChannel.js:10]: Yes

**Pull request merged**